zuul.d/content_provider.yaml

```diff
@@ -16,18 +16,35 @@
       - ^kuttl-test.yaml$
     required-projects:
       - opendev.org/zuul/zuul-jobs
+      - openstack-k8s-operators/ansible-microshift-role
```

Comment:
in content provider job, these projects ansible-microshift-role, data-plane-adoption, edpm-image-builder, edpm-ansible, lib-common, os-diff are not needed. We can remove that.